friends/src/actions/index.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+export const FETCHING = 'FETCHING';
+export const FETCHED = 'FETCHED';
+export const ERROR = 'ERROR';
+export const ADDING = 'ADDING';
+export const ADDED = 'ADDED';
+export const DELETED = 'DELETED';
+export const DELETING = 'DELETING';
+
+export const fetching = () => {
+
+    return (dispatch) => {
+      dispatch({type: FETCHING});
+      axios.get('http://localhost:5000/api/friends')
```

Comment:
I'd recommend creating a variable for your API URLs since it's something that you will be using quite often within your code moving forward. 
After creating a variable such as `const URL = 'http://localhost:5000/api/friends';`, instead of `axios.get('http://localhost:5000/api/friends')` you could have `axios.get("${URL}/...");`, and you could then use the same throughout the rest of your code such as below with the POST request. 
* Please note that the string literal won't display correctly here, so the quotes should be replaced with backticks.